.circleci/config.yml

```diff
@@ -112,6 +112,28 @@ jobs:
           build-dir: docs/manual/book
           ssh-fingerprints: "9d:06:af:45:32:f9:71:b7:57:00:81:7f:70:6b:d1:49"
 
+  Deploy Docker:
+    docker:
+      - image: circleci/python
+    steps:
+      - checkout
+      - setup_remote_docker
+
+      - run:
+          name: Build container image
+          command: |
+            docker build -t uniffi-ci:build -f docker/Dockerfile-build .
+      - run:
+          name: Push to Dockerhub
+          command: |
+            cd docker
+            if [ -n "${CIRCLE_TAG}" ]; then
+              # Strip the `docker-` prefix of the tag
+              TAG="${CIRCLE_TAG#docker-}"
+              ./deploy-dockerhub.sh "$TAG"
+              ./deploy-dockerhub.sh "latest"
```

Comment:
I wonder if there'd ever be a case where the new tag is *not* actually the new "latest"

Reply:
Honestly for that Docker image I would keep it simple (not even use `X.Y.Z` versioning, just strictly incrementing the version `X -> X+1`.
Rollbacks should be easy enough.